go_modules/spec/fixtures/projects/transferred_private_package/go.sum

```diff
@@ -0,0 +1,573 @@
+bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690/go.mod h1:Ulb78X89vxKYgdL24HMTiXYHlyHEvruOj1ZPlqeNEZM=
```

Comment:
This is a hot mess, because I yanked this from an actually failing (public) repo, but since this project doesn't build, I can't `go mod tidy` this go.sum file. Leaving it as is seemed better than trying to manually clean it up.